### PR TITLE
Refactor the function of retriever to improve code style

### PR DESCRIPTION
**Pull request by ShengguangXiao**
## Description

Refactor the function of retriever to improve code style

Refactor the function of retriever to improve code style

## Review Suggestion

Check the Retriever.cpp

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [x] local machine test

- [ ] small-scale cloud test

### CODE REVIEW

File: src/libNode/Node.cpp [hunk 1]
```diff
@@ -282,13 +282,11 @@ bool Node::StartRetrieveHistory(bool& wakeupForUpgrade) {
   }
 
   /// Retrieve DS blocks
-  bool ds_result;
-  m_retriever->RetrieveBlockLink(ds_result, wakeupForUpgrade);
+  bool ds_result = m_retriever->RetrieveBlockLink(wakeupForUpgrade);
```
Comment (ShengguangXiao) [inline src/libNode/Node.cpp]: @KaustubhShamshery Here the comments write to retrieve DS blocks, but function call is to retrieve block link, is there any problem here?

Comment (KaustubhShamshery) [reply src/libNode/Node.cpp]: I think the comment needs to be changed, can you incorporate this in your PR?

Comment (ShengguangXiao) [reply src/libNode/Node.cpp]: Ok, can

Comment (KaustubhShamshery) [reply src/libNode/Node.cpp]: Thanks !!

**Pull request merged**